src/components/nodes/boundaryEvent/boundaryEvent.vue

```diff
@@ -50,50 +50,28 @@ export default {
         .filter(model => model.component)
         .find(model => taskIds.includes(model.component.node.type));
     },
-    updateBoundaryShape(dashLength) {
+    setShapeStrokeDasharry(dashLength) {
```

Comment:
@chipit24 is `Dasharry` a typo? From looking at the body of the method it seems to be a misspelling of `Dasharray`.

Even if you correct the typo, I'm not sure if this is a good name. Can you think of a better one?